base/cvd/cuttlefish/host/commands/cvd/server_command/stop.cpp

```diff
@@ -0,0 +1,304 @@
+/*
+ * Copyright (C) 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "host/commands/cvd/server_command/stop.h"
+
+#include <sys/types.h>
+
+#include <functional>
+
+#include <android-base/file.h>
+#include <android-base/parseint.h>
+#include <android-base/scopeguard.h>
+
+#include "common/libs/fs/shared_buf.h"
+#include "common/libs/fs/shared_fd.h"
+#include "common/libs/utils/contains.h"
+#include "common/libs/utils/files.h"
+#include "common/libs/utils/result.h"
+#include "common/libs/utils/subprocess.h"
+#include "cuttlefish/host/commands/cvd/cvd_server.pb.h"
+#include "host/commands/cvd/common_utils.h"
+#include "host/commands/cvd/group_selector.h"
+#include "host/commands/cvd/instance_manager.h"
+#include "host/commands/cvd/interruptible_terminal.h"
+#include "host/commands/cvd/selector/selector_constants.h"
+#include "host/commands/cvd/server_command/host_tool_target_manager.h"
+#include "host/commands/cvd/server_command/server_handler.h"
+#include "host/commands/cvd/server_command/subprocess_waiter.h"
+#include "host/commands/cvd/server_command/utils.h"
+#include "host/commands/cvd/types.h"
+
+namespace cuttlefish {
+namespace {
+
+constexpr char kSummaryHelpText[] =
+    "Run cvd stop --help for command description";
+
+}  // namespace
```

Comment:
`CvdStopCommandHandler` could be in the unnamed namespace as well, only `NewCvdStopCommandHandler` needs to be outside.

Reply:
ACK.

I just copied everything from generic.cpp and deleted the things not relevant to the stop command. I think moving things to the empty namespace should include all other commands and be done in a separate PR.